### PR TITLE
Refresh README.md

### PR DESCRIPTION
**Pull request by tofu-rocketry**
- Use different levels of heading.
- Add build status icon.
- Remove reference to re-implementation of legacy aspects.
- Replace logo with new set.
- Update funding message.
- Wordify 'docs' to 'documentation'.
- Linkify INSTALL.md reference.

### CODE REVIEW

**Comment by jrha** [comment]
Why remove the line breaks between images, it's much less readable.

Also it might be an idea to use the [GFM image syntax](https://github.github.com/gfm/#images).

File: README.md [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-Documentation 
-=============
-* [GOCDB Executive Summary](https://wiki.egi.eu/w/images/d/d3/GOCDB5_Grid_Topology_Information_System.pdf)
-* [EGI GOCDB wiki landing page](https://wiki.egi.eu/wiki/GOCDB) 
-* [GOCDB Documentation index](https://wiki.egi.eu/wiki/GOCDB_Documentation_Index)
-* [User docs](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation) 
-* INSTALL.md for installation 
+<img alt="STFC logo" src="htdocs/web_portal/img/STFCMediumColourTrans_crop.png" height=57/> <img alt="EU flag" src="htdocs/web_portal/img/eu_flag_yellow_low_150.png" height=57 /> <img alt="EGI logo" src="htdocs/web_portal/img/egi_logo.jpg" height=57 /> <img alt="EOSC-hub logo" src="htdocs/web_portal/img/eosc-hub-v-web_150.png" height=57 />
+
+GOCDB is provided by <a class="Licence_Link" href="https://stfc.ukri.org/">
```
Comment (jrha) [inline README.md]: Fairly sure that the class won't be do anything here, so the markdown syntax would be more readable:
`[STFC](https://stfc.ukri.org/)`

**Comment by tofu-rocketry** [comment]
> Why remove the line breaks between images? It's much less readable.

Yes, it is less readable but GitHub renders them on separate lines otherwise.

> Also it might be an idea to use the [GFM image syntax](https://github.github.com/gfm/#images).

I wanted to be able to control the height of the images to make them all the same. GFM doesn't seem to support that.

File: README.md (hunk 1 above)
Comment (tofu-rocketry) [reply README.md]: Yup, but I just left it as it was. Can fix it as I'm here anyway.

**Comment by jrha** [comment]
> Yes, it is less readable but GitHub renders them on separate lines otherwise.
Wrapping them in a `<span>` would accomplish the same thing!

File: README.md [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
-* [User docs](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation) 
-* INSTALL.md for installation 
+<span>
+  <img alt="STFC logo" src="htdocs/web_portal/img/STFCMediumColourTrans_crop.png" height=57/>
+  <img alt="EU flag" src="htdocs/web_portal/img/eu_flag_yellow_low_150.png" height=57 />
+  <img alt="EGI logo" src="htdocs/web_portal/img/egi_logo.jpg" height=57 />
+  <img alt="EOSC-hub logo" src="htdocs/web_portal/img/eosc-hub-v-web_150.png" height=57 />
+</span>
+
+GOCDB is provided by [STFC](https://stfc.ukri.org/) for [EGI](http://www.egi.eu), co-funded by [EGI.eu](http://www.egi.eu) and [EOSC-hub](http://go.egi.eu/eng). Licensed under the [Apache 2 License](ttp://www.apache.org/licenses/LICENSE-2.0).
```
Comment (jrha) [inline README.md]: **h**ttp

Comment (tofu-rocketry) [reply README.md]: Sorted.

**Pull request merged**